code/game/objects/items/devices/camera_bug.dm

```diff
@@ -87,6 +88,7 @@
 		return 1
 	to_chat(user, "<span class='notice'>You stealthily place \the [src] onto \the [A].</span>")
 	active = TRUE
+	placed_turf = A
```

Comment:
Like the type of the var implies, this is not necessarily a turf, so the var name is misleading. Also, keeping a reference to the thing the bug is on will prevent the thing from properly being deleted.

Reply:
I'd be willing to change the name, but keeping a reference was the only way, at least that I found, that would make taking photos through a camerabug that had been dropped into another object actually go through.

Reply:
Just did some deeper testing, and it seems like if you attach a bug to something, take it off, and bring it with you, you can still take pictures of what it was attached to.